dlc-wallet-test/src/test/scala/org/bitcoins/dlc/wallet/RescanDLCTest.scala

```diff
@@ -101,4 +107,73 @@ class RescanDLCTest extends DualWalletTestCachedBitcoind {
       postStatus <- getDLCStatus(wallet)
     } yield assert(postStatus.state == DLCState.RemoteClaimed)
   }
+
+  it must "properly rescan when a dlc is remotely claimed" in { params =>
```

Comment:
This test case passes on master, so either 

1. The test case isn't accurately reproducing the bug
2. Something else in master fixed this bug


I'm unsure which one is happening.